[2024-12-03 05:49:04,158] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: KPIs_for_1_Minutes.Hello_world 2024-12-03T05:48:00+00:00 [queued]>
[2024-12-03 05:49:04,196] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: KPIs_for_1_Minutes.Hello_world 2024-12-03T05:48:00+00:00 [queued]>
[2024-12-03 05:49:04,197] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-12-03 05:49:04,198] {taskinstance.py:1088} INFO - Starting attempt 1 of 3
[2024-12-03 05:49:04,198] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-12-03 05:49:04,261] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): Hello_world> on 2024-12-03T05:48:00+00:00
[2024-12-03 05:49:04,267] {standard_task_runner.py:52} INFO - Started process 1708 to run task
[2024-12-03 05:49:04,271] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'KPIs_for_1_Minutes', 'Hello_world', '2024-12-03T05:48:00+00:00', '--job-id', '431', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/Kpi_dags/main.py', '--cfg-path', '/tmp/tmpxqfvq4ma', '--error-file', '/tmp/tmp3yk_bbb4']
[2024-12-03 05:49:04,273] {standard_task_runner.py:77} INFO - Job 431: Subtask Hello_world
[2024-12-03 05:49:04,387] {logging_mixin.py:104} INFO - Running <TaskInstance: KPIs_for_1_Minutes.Hello_world 2024-12-03T05:48:00+00:00 [running]> on host 4dee9495e4cc
[2024-12-03 05:49:04,573] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Rule Manager
AIRFLOW_CTX_DAG_ID=KPIs_for_1_Minutes
AIRFLOW_CTX_TASK_ID=Hello_world
AIRFLOW_CTX_EXECUTION_DATE=2024-12-03T05:48:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-12-03T05:48:00+00:00
[2024-12-03 05:49:04,574] {main.py:51} INFO - Hello World
[2024-12-03 05:49:04,576] {main.py:52} INFO - time taken to connect to kpi_main_table db : 0:00:00.317975
[2024-12-03 05:49:04,577] {main.py:53} INFO -    id    kpi_name  ... last_failure_time version_id
0   2  Wind Speed  ...              None          1
1   3              ...              None          1

[2 rows x 28 columns]
[2024-12-03 05:49:04,610] {python.py:151} INFO - Done. Returned value was: None
[2024-12-03 05:49:04,638] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=KPIs_for_1_Minutes, task_id=Hello_world, execution_date=20241203T054800, start_date=20241203T054904, end_date=20241203T054904
[2024-12-03 05:49:04,760] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-03 05:49:04,778] {local_task_job.py:151} INFO - Task exited with return code 0
